tests/testthat/test-seq_linter.R

```diff
@@ -49,6 +49,33 @@ test_that("finds 1:length(...) expressions", {
     rex("dim(...)", anything, "Use seq_len"),
     linter
   )
+
+  expect_lint(
+    "function(x) { mutate(x, .id = 1:n()) }",
+    rex("n()", anything, "Use seq_len"),
+    linter
+  )
+
+  # `dplyr::n()` takes no arguments, so the message shouldn't have `n(...)`
+  expect_error(expect_lint(
```

Comment:
not necessary, already implied by the first test

Reply:
(btw I think testthat offers expect_failure() for expecting failed expectations, and expect_success for the reverse)